go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/nats-io/nats-server/v2
 
 go 1.20
 
+replace github.com/nats-io/nats.go => github.com/nats-io/nats.go v1.33.1-0.20240216092839-2c1498a72b57
```

Comment:
Currently that commit is still on a branch, so I wasn't sure if it would get merged in as-is or rebased or so first. If you're happy with the API shape then https://github.com/nats-io/nats.go/pull/1554 can get merged and I can remove the `replace`.

Reply:
Ah did not notice it was on a branch..

Reply:
But I think we can pull it in from a branch without a replace directive no?